lms/djangoapps/lti_provider/tests/test_views.py

```diff
@@ -282,3 +282,6 @@ def get_response(self):
         )
         SignatureValidator.verify = MagicMock(return_value=True)
         return self.client.post(lti_launch_url, data=LTI_DEFAULT_PARAMS)
+
+    def test_fail_unenrolled_student(self):
+        pass
```

Comment:
To make sure we don't break LTI for unenrolled students again, can you:

1. rename this test to `test_unenrolled_student` (both here and in `RenderXBlockTestMixin`)
2. update the implementation of this test to verify that an LTI unenrolled student can successfully access the endpoint.  Something like:
```
        self.setup_course()
        self.setup_user(admin=False, enroll=False, login=True)
        self.verify_response()
```

Thanks.